benchmarks/conftest.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
move this file to a separate PR and merge it first. This should solve the error you are seing about missing --rank

Reply:
Done thanks